app/services/applier/greenhouse_form_filler.rb

```diff
@@ -6,11 +6,6 @@ class GreenhouseFormFiller < FormFiller
 
```

Comment:
Difference between gh_form_filler & greenhouse_form_filler is old vs new formats presumably - can we highlight this somewhere

Reply:
Commented